app/templates/my-projects/assignment/hearing/add.hbs

```diff
@@ -6,7 +6,7 @@
       <h3 class="tiny-margin-bottom">
         <LinkTo
           @route="show-project"
-          @model={{project}}
+          @model={{model}}
```

Comment:
Seem like this should be `{{model.project}}`, since `model` is now an assignment but is related to project

Reply:
🤷‍♂ I did this days ago, before assignments were a thing. I'll look at it again… 